src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -118,28 +118,24 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/k8s-
   >
     To deploy the Kubernetes integration with [OpenShift](https://learn.openshift.com):
 
-    1. Add the `<>{'<release_name>'}</>-newrelic-infrastructure` service account to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
+    1. Add the service accounts used by the integration to your privileged [Security Context Constraints](https://docs.openshift.com/enterprise/3.0/admin_guide/manage_scc.html):
 
        ```
-       oc adm policy add-scc-to-user privileged \
-       system:serviceaccount:<var><namespace></var>:<var><release_name></var>-newrelic-infrastructure
+       oc adm policy add-scc-to-user privileged system:serviceaccount:<var><namespace></var>:<var><release_name></var>-newrelic-infrastructure
+       oc adm policy add-scc-to-user privileged system:serviceaccount:<var><namespace></var>:<var><release_name></var>-newrelic-infrastructure-controlplane
+       oc adm policy add-scc-to-user privileged system:serviceaccount:<var><namespace></var>:<var><release_name></var>-kube-state-metrics
+       oc adm policy add-scc-to-user privileged system:serviceaccount:<var><namespace></var>:<var><release_name></var>-newrelic-logging
+       oc adm policy add-scc-to-user privileged system:serviceaccount:<var><namespace></var>:<var><release_name></var>-nri-kube-events
```

Comment:
note that the formatting for the `<var><namespace></var>` seems to be broken, can doc team fix it? 😄